src/mainwindow.cpp

```diff
@@ -401,6 +401,7 @@ void MainWindow::setupMainWindow()
     ui->searchEdit->setFocus();
     QTimer::singleShot(16, ui->searchEdit, &QTextEdit::clearFocus);
 #endif
+    this->setWindowIcon(QIcon(QStringLiteral(":images/notes_icon.ico")));
```

Comment:
```suggestion
    setWindowIcon(QIcon(QStringLiteral(":images/notes_icon.ico")));
```
I realize we have a lot of needless `this->` in the file already, but at least I'd prefer to remove them instead of add more. :)